template-parts/homepage-landing.php

```diff
@@ -9,21 +9,21 @@
 
 <script type="text/javascript">
 	// Fetch JSON data for dynamic slider numbers
-	var fetch1 = jQuery.getJSON("<?php echo get_template_directory_uri(); ?>/data/cdr_compressed.json", function (cdrData) {
-		var cdrStartingYear = cdrData.meta.lookups.year[0];
-		var cdrTotalRecords = cdrData.meta.num_records;
+	var fetch1 = jQuery.getJSON("https://s3.amazonaws.com/tji-compressed-data/cdr_slider.json", function (cdrData) {
```

Comment:
Hey @judan @aidenyang -- I think I wasn't super clear about the data change request for the homepage. It would be much faster to have all the data in one file, so the slider wouldn't have to make 3 separate requests for 6 simple numbers. If all the data could be combined into one json file -- and only one request -- that would be best.